nixos/modules/module-list.nix

```diff
@@ -916,6 +916,7 @@
   ./services/misc/redlib.nix
   ./services/misc/redmine.nix
   ./services/misc/renovate.nix
+  ./services/misc/ringboard.nix
```

Comment:
Could you add a release note to `nixos/doc/manual/release-notes/rl-2511.section.md`

Reply:
Like this?
```
- [ringboard](https://github.com/SUPERCILEX/clipboard-history), a fast, efficient, and composable clipboard manager for Linux. Available for x11 as [services.ringboard](#opt-services.ringboard.x11.enable) and for wayland as [services.ringboard](#opt-services.ringboard.wayland.enable).
```